modules/nf-core/wfmash/meta.yml

```diff
@@ -30,7 +30,7 @@ input:
       description: Optional inpute file in PAF format to derive the precise alignments for.
       pattern: "*.{paf}"
   - query_self:
-      type: val
+      type: boolean
       description: If set to true, the input FASTA will also be used as the query FASTA.
       pattern: "{true, false}"
```

Comment:
If it's a nextflow variable, a pattern is not required, right?

Reply:
You are right. I updated the PR.